docs/source/pages/howToWorkWithPlatform/mainApplication/templateList/templateManagement/documentManagementOnTemplate/addDocument/fields/duplicate/duplicate.rst

```diff
@@ -0,0 +1,68 @@
+============
+Duplicate Field
+============
+
+.. toctree::
+
+Duplicate field copies the value of the parent field.
+Duplicate field is available only when the template contains at least one field to which you can assign a duplicate.
+Allowed fields for duplicating: text, number, currency, date and time, dictionary, yes/no.
+
+Create Duplicate Field
+===================
+
+1. Navigate to https://staging.whitedoc.space/template/form/editor
```

Comment:
remove URL, we have to write navigate to platform, open template list, create template and etc.

It's necessary to don't show links

Reply:
Also you can write: Open template for edition and open document for edition